.travis.yml

```diff
@@ -1,23 +0,0 @@
-language: node_js
-node_js:
-  - "6"
-  - "8"
-  - "10"
-cache:
-  yarn: true
-  directories:
-    - node_modules
-before_install:
-  - npm install --global yarn
-script:
-  - npm run lint
-  - npm test
-before_script:
-  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
-  - chmod +x ./cc-test-reporter
-  - ./cc-test-reporter before-build
-after_script:
-  - npm run cover
-  - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
```

Comment:
This is actually broken and it is silently outputting an error. There is https://github.com/apiaryio/api-elements.js/issues/20 to track adding coverage.